services/core/package.json

```diff
@@ -52,7 +52,8 @@
     "split": "^1.0.1",
     "stack-trace": "0.0.10",
     "string-similarity": "^4.0.4",
-    "tar": "^6.1.9"
+    "tar": "^6.1.9",
+    "zlib": "^1.0.5"
```

Comment:
NodeJS by default has the `zlib` (https://nodejs.org/api/zlib.html#zlib) module available that also offers the `createUnzip` method. Kindly remove this dependency and re-test the implementation. I can confirm, the download utility works on my machine even without installing this dependency.